app/src/main/res/drawable/ic_vpn_key.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
ic_vpn_key ?

Missed renaming? :)

Reply:
No I am always leaving the official names for the files, so you can easily see if you add an icon that already exists.